oeps/processes/oep-0062-proc-working-groups.rst

```diff
@@ -0,0 +1,218 @@
+
+Forming
+-------
+
+To form a working group, any community member can share an announcement describing:
+
+* the new group's name,
+* its purpose,
+* how to get involved in the new group, and
+* optionally: who they suggest as the initial chairperson.
+
+If the announcement generates sufficient interest, then the working group is formed. If not already selected, their first order of business should be to select a chairperson. Once selected, the new chairperson should publicise the new group's details as described above.
+
+Changing
+--------
+
+At any time, by consensus amongst involved groups, working groups may merge together, split apart into multiple working groups, select a new chairperson, change their structure or processes, and so on. Working groups are encouraged to announce any significant changes to the broader community.
+
+Dissolution
+-----------
+
+Eventually, it may make sense to dissolve the working group for various reasons, such as inactivity, redundancy, lack of participation, or resolution of the group's purpose. At this point, consensus among the remaining working group members is sufficient to commence dissolution of the group. 
+
+If the working group was responsible for handling any community processes, the remaining members are encouraged to seek new stewards for those responsibilities; if they cannot, they should notify the group of Open edX Core Contributors, who can assist in filling the roles or escalate further. If the working group was named maintainer of any project components, then the remaining members should follow the `maintainership transfer process <https://docs.openedx.org/projects/openedx-proposals/en/latest/processes/oep-0055/decisions/0002-maintainership-transfer-process.html>`_.
+
+One of the remaining working group members should update the working group's home page to reflect the dissolution. They should announce the dissolution and any related responsibility transfers to the community.
+
+If there seem to be no remaining members in working group and the last chairperson cannot be reached, then the group is *de facto* dissolved. Any Core Contributor is encouraged to take the above steps in lieu of the abandoned working group.
+
+Rejected Alternatives
+*********************
+
+No Formal Guidelines
+====================
+
+Without an OEP, working groups would still continue operating as they are today, possibly without any issues. So, we could have forgone this OEP.
+
+However, we believe that when critical processes left undocumented, it becomes harder for newcomers to aquaint themselves with a project. Furthermore, undocumented proccesses work because certain community members happen to remember them; as those community members retire or move on to other projects, their knowledge can be lost. Finally, this OEP does add certain new guidelines that we believe are important; for example, the process to step down as a chairperson or disssolve a group are new.
```

Comment:
Yes. This was extremely confusing to approach as a newer person in the community. I definitely think it needs to be more explicit.

Reply:
That is great feedback to hear, thank you. I was worried writing this that I might just be stating the obvious, to the benefit of nobody.